modules/social_features/social_comment/social_comment.tokens.inc

```diff
@@ -317,7 +317,7 @@ function _social_comment_get_summary($text) {
   $summary = preg_replace('/\n|\r|\t/m', ' ', $summary);
   $max_length = 280;
 
-  if (mb_strlen($summary) > $max_length) {
+  if (is_string($summary) && (mb_strlen($summary) > $max_length)) {
```

Comment:
[preg_replace](https://www.php.net/manual/en/function.preg-replace) has return type of `string|array|null`, so phpstan suggests us to check for returning values.

Reply:
Okay yes, but will the rest of our code work is `preg_replace` return `NULL`? In what scenario's does `preg_replace` return `NULL` or `array`? Does that happen in the way we're using it?

Could we instead use an `assert(is_string($summary), "<expectation>")` beneath `preg_replace`? Making it part of the if statement suggests that it's part of our business logic or what we expect scenarios where `$summary` is not a string (rather than it being a quirk of PHP).